pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -66,13 +66,15 @@
   private final ZkHelixPropertyStore<ZNRecord> _propertyStore;
   private final LiveInstancesChangeListenerImpl _liveInstancesListener;
 
-  private static final Logger LOGGER = LoggerFactory.getLogger("HelixBrokerStarter");
+  private static final Logger LOGGER = LoggerFactory.getLogger(HelixBrokerStarter.class);
 
   private static final String ROUTING_TABLE_SELECTOR_SUBSET_KEY =
       "pinot.broker.routing.table.selector";
 
   public HelixBrokerStarter(String helixClusterName, String zkServer, Configuration pinotHelixProperties)
       throws Exception {
+    LOGGER.info("Starting Pinot broker...");
```

Comment:
[minor] I suggest removing ellipsis. It looks more professional that way. Ellipsis makes better sense on interactive console